### PR TITLE
Add #TinyGo Slack to the README

### PR DESCRIPTION
**Pull request by justinclift**
Addresses #54.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -142,6 +142,15 @@ Non-goals:
 Documentation is currently maintained on a [dedicated ReadTheDocs
 page](https://tinygo.readthedocs.io/en/latest/).
 
+## Slack channel
```
Comment (aykevl) [inline README.markdown]: I wonder whether "Slack channel" is a good name? Perhaps something like "Getting help" is better, as it is more general - maybe we'll add a forum in the future (or subreddit, or something like that).
I have looked around a bit but such a section is surprisingly rare in a README, so not sure what other projects call it.

Comment (justinclift) [reply README.markdown]: Excellent.  I was struggling with a section name to use.  "Getting help" does sound better, and leaves room for future expansion. :smile:

**Comment by deadprogram** [comment]
Looks ready to merge.

**Comment by justinclift** [comment]
Oops yes, forgot to mention it.  It should now be ready. :smile:

**Comment by aykevl** [comment]
Thank you! Merged in 586d3b33b108495ece60e54203f769511538cfe4.

**Pull request closed without merging**